api/exceptions.py

```diff
@@ -35,13 +35,20 @@ def __init__(self):
         super().__init__(status_code=401, detail="The access token has expired")
 
 
-class AuthorizationIssueException(HTTPException):
+class AuthenticationIssueException(HTTPException):
     """Exception raised when Nexus throws authorization error"""
 
     def __init__(self):
         super().__init__(status_code=401, detail="The user does not have access in the content")
```

Comment:
I would think different detail messages could clarify the distinction between 401 and 403

Reply:
Thats a good point, I modified the message of 401